wasm-engines/docker/wasmi.Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM jwasinger/llvm-10:1.0
+
+# install rust
+RUN curl https://sh.rustup.rs -sSf | \
+    sh -s -- --default-toolchain 1.42.0 -y && . $HOME/.cargo/env
+ENV PATH=/root/.cargo/bin:$PATH
+
+RUN git clone --single-branch --branch bench-time https://github.com/ewasm-benchmarking/wasmi.git --recursive
+RUN cd wasmi && cargo test --release
```

Comment:
Why are you running test??

You should only do `cargo build --release` as that is the release binary and it will not try to download wabt.

Reply:
This is a holdover from the previous Dockerfile that I didn't catch.  Have fixed it now (testing the changes locally before I push).